examples/ws2812/main.go

```diff
@@ -1,8 +1,8 @@
-// Connects to an WS2812 RGB LED strip with 10 LEDS, such as
-// on an Adafruit Circuit Playground Express board.
+// Connects to an WS2812 RGB LED strip with 4 LEDS
```

Comment:
no longer true, it should be 10 LEDs. Also, not only on digispark

Reply:
Good point, fixed.